java/src/jmri/jmrix/mrc/MrcSystemConnectionMemo.java

```diff
@@ -195,7 +195,7 @@ public MrcClockControl getClockControl() {
     /*public MrcConsistManager  getConsistManager() { return consistManager; }*/
 
     protected ResourceBundle getActionModelResourceBundle() {
-        return ResourceBundle.getBundle("jmri.jmrix.mrc.MrcActionListBundle"); //IN18N
+        return ResourceBundle.getBundle("jmri.jmrix.mrc.MrcBundle"); //IN18N
```

Comment:
This needs to be comments ```// NOI18N``` to indicate that the String literal "jmri.jmrix.mrc.MrcBundle" *is not* to be translated.